### ISSUE TITLE
Vertical cube transition

### ISSUE THREAD

**Issue by aksonov** [tlackemann/react-native-cube-transition#1]
Is it possible to do vertical cube transition with this module? Thanks

**Comment by tlackemann** [comment]
At the moment no. I don't have plans to integrate this either but am open to taking a pull request.

**Comment by neo125874** [comment]
@tlackemann would you plan to have the android version?

**Comment by tlackemann** [comment]
@neo125874 I would be open to accepting/reviewing a PR but my time has not allowed me to work on this in some time

**Comment by neo125874** [comment]
@tlackemann many many thanks for your ios work, any keywords for android native code, maybe i could do some research?

**Comment by tlackemann** [comment]
@neo125874 Certainly! I know Android has "cube transition" built in although it's been awhile since I've touched any React Native code unfortunately so I couldn't tell you exactly how it works. I know the package manipulates the iOS `UIView`, you'd need to find the equivalent on Android

**Comment by neo125874** [comment]
@tlackemann your ios work really nice, I try some pure js code example but not working on Android, thanks for your fragment keyword, I almost forgot the native android component about that.
